src/Data/Fin/Substitution/Lemmas.agda

```diff
@@ -346,6 +346,14 @@ record Lemmas₄ {ℓ} (T : Pred ℕ ℓ) : Set ℓ where
     lookup ρ x / wk          ≡⟨ cong₂ _/_ (sym var-/) refl ⟩
     var x / ρ / wk           ∎
 
+  weaken-↑ : ∀ {k n} t {ρ : Sub T k n} → weaken t / (ρ ↑) ≡ weaken (t / ρ)
+  weaken-↑ t {ρ} = begin
```

Comment:
Please could you layout the reasoning proofs as specified in the [style guide](https://github.com/agda/agda-stdlib/blob/master/notes/style-guide.md).